content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_resupply.rst

```diff
@@ -0,0 +1,260 @@
+======================
+Resupply subcontractor
+======================
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |POs| replace:: :abbr:`PO (Purchase Orders)`
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In manufacturing, subcontracting is the process of a company engaging a third-party manufacturer, or
+subcontractor, to manufacture products that are then sold by the contracting company.
+
+In Odoo, the *Resupply Subcontractor on Order* route is used to deliver the necessary components for
+a subcontracted product to the subcontractor, each time a purchase order (PO) for that product is
+confirmed.
```

Comment:
Optional: Consider adding more information here (maybe in a **Note** below this section) that explicitly states that this is different than _Dropship to Subcontractor_, which leverages a VENDOR location to send components vs. leveraging OUR internal location to send components that we have on-hand. A lot of the steps are similar between the two resupply methods, so I could see people getting confused here.